src/ray/object_manager/object_manager.cc

```diff
@@ -22,6 +22,19 @@
 
 namespace asio = boost::asio;
 
+DEFINE_stats(num_chunks_received_total, "Number object chunks received.", (), (),
+             ray::stats::GAUGE);
+DEFINE_stats(num_chunks_received_failed,
+             "Number of object chunks failed to be transferred.", (), (),
+             ray::stats::GAUGE);
```

Comment:
Shouldn't the errors be labels on num_chunks_received, instead of defining separate stats?